frontend/packages/dev-console/src/components/topology/components/nodes/BaseNode.scss

```diff
@@ -59,4 +55,10 @@
       opacity: 1;
     }
   }
-}
\ No newline at end of file
+}
+
+.odc-m-drag-active {
+  .odc-base-node {
+    cursor: grab;
+  }
+}
```

Comment:
We do this in other scss files instead:

```
  .odc-m-drag-active & {
    cursor: grab;
  }
```

But i think it should be:
```
  &.is-dragging {
    cursor: grab;
  }
```

Reply:
The first is for when an item is being dragged, the second for when an item is dragged over an item (like a move/create connector) to avoid flicker should the cursor move off the dragged item momentarily.